### ISSUE TITLE
RRmpWald with formula option can break

### ISSUE THREAD

**Issue by nulloa** [ABS-dev/PF#47]
When using the formula option in  `RRmpWald`, if there are no unaffected cases in the controls, the function spits an error.

Here is a reproducible example using the `New` dataset loaded by default with `PF`:
```
RRmpWald(pos ~ tx + cluster(cage), New, compare = c('vac', 'con'))

# PF 
# 95% interval estimates
#
#    PF    LL    UL 
# 0.550 0.183 0.752 
```

This works fine because there are unaffected in the controls, but if we change this, it will break:
```
New[which(New$tx=="con"),"pos"] <- 1

RRmpWald(pos ~ tx + cluster(cage), New, compare = c('vac', 'con'))
Error in `[.default`(xtable, c("af", "un"), c("af", "un")) : 
  subscript out of bounds
```

This is an error referring to this line:
`xtable <- table(tbl[, compare[1]], tbl[, compare[2]])`  in the `PF:::.twoby()` function. 
Basically, what needs to be fixed is there needs to be an additional statement filling out the frequency table when there are no affected controls.

However, there is a workaround. The vector option still works:
```
require(magrittr)
thistable <- New %>%
  tidyr::spread(tx, pos) %>%
  dplyr::mutate(vac = factor(vac, levels = 1:0),
    con = factor(con, levels = 1:0)) %>%
  with(., table(vac, con)) 
thistable
#   con
#vac  1  0
#  1  9  0
#  0 17  0

RRmpWald(x = as.vector(thistable))

PF 
95% interval estimates

   PF    LL    UL 
0.654 0.396 0.802 

Warning message:
In q * sqrt(varlogR) :
  Recycling array of length 1 in vector-array arithmetic is deprecated.
  Use c() or as.vector() instead.
```

You still do get the warning message mentioned in #5 but it works.

**Comment by TFKentUSDA** [comment]
Closed with commit 89b0d458c23b7
